README.md

```diff
@@ -33,25 +31,27 @@ For understanding the result, please see the `autocannon-compare` [documentation
 
 ## Getting started
 
-- `npm install && npm install -g`
+- `npm install`
 
 The easiest way to try this out is to run:
 
 - `npx autocannon-ui`
-- browse to [`http://localhost:3000`](http://localhost:3000)
+- browse to localhost -- the port is displayed when running this command
 
 ## Development
 
 To easily develop the packages of this repo you can execute:
 
 ```sh
-npm start (npx autocannon-ui after npm install -g)
+npm install
+npm start
 ```
 
 This will run:
 
 - `autocannon-frontend` build in watch mode so you can change the React components and see the result immediately
 - `autocannon-backend` in standalone mode to have a server running
+- the port will be displayed where the application is running
```

Comment:
I would suggest that when running in development, we run on a specific port instead, and not open any browser windows automatically